scripts/build_project.sh

```diff
@@ -0,0 +1,43 @@
+# Author: Teghveer Singh Ateliey
+# Date: 2024-10-30
+
+#!/bin/bash
+
+# Check args
+if [ "$#" -ne 4 ]; then
+    echo "Usage: $0 <commit-hash> <repo-path> <project> <platform>"
+    exit 1
+fi
+
+# Assign inputs to variables
+COMMIT_HASH=$1
+REPO_PATH=$2
+PROJECT=$3
+PLATFORM=$4
+
+# Checks if directory exists
+if [ ! -d "$REPO_PATH" ]; then
+    echo "Error: Repository path '$REPO_PATH' does not exist."
+    exit
+fi
+
+# Navigate to the racecar/firmware directory path
+cd "$REPO_PATH" || exit
+
+# Activate the Python virtual environment
+source .env/Scripts/activate
```

Comment:
this is not cross platform, venvs look different on linux vs mac vs windows. You can remove this part of the command and just assume that the environment is already active